pkg/deploy/taskdir/definitions/def_0_3.go

```diff
@@ -1196,6 +1195,35 @@ func (d Definition_0_3) addParametersToUpdateTaskRequest(ctx context.Context, re
 			return errors.Errorf("unknown parameter type: %s", pd.Type)
 		}
 
+		switch reflect.ValueOf(pd.Default).Kind() {
+		case reflect.Map:
+			m, ok := pd.Default.(map[string]interface{})
+			if !ok {
+				return errors.Errorf("unhandled default: %v", pd.Default)
+			}
+			if configName, ok := m["config"]; !ok {
+				return errors.Errorf("unhandled default: %v", pd.Default)
+			} else {
+				param.Default = map[string]interface{}{
+					"__airplaneType": "configvar",
+					"name":           configName,
+				}
+			}
+		case reflect.String, reflect.Bool, reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Int64,
+			reflect.Uint, reflect.Uint8, reflect.Uint16, reflect.Uint32, reflect.Uint64,
+			reflect.Float32, reflect.Float64:
+			if pd.Type == "configvar" {
+				param.Default = map[string]interface{}{
+					"__airplaneType": "configvar",
+					"name":           pd.Default,
+				}
+			} else {
+				param.Default = pd.Default
+			}
+		default:
+			return errors.Errorf("unhandled default value: %T", pd.Default)
```

Comment:
Bit more clear for end users:

```suggestion
			return errors.Errorf("unsupported type for default value: %T", pd.Default)
```